app/routers/api/paths.py

```
@@ -292,6 +292,27 @@ def create_path(path: schemas.MonitoredPathCreate, db: Session = Depends(get_db)
+        notification_service.dispatch_event_sync(
+            db=db,
+            event_type=NotificationEventType.PATH_CREATED,
+            event_data=PathCreatedData(
+                path_id=db_path.id,
+                path_name=db_path.name,
+                source_path=db_path.source_path,
+                operation_type=db_path.operation_type.value,
+                created_by=None,  # TODO: Add auth context when implemented
+            ),
+        )
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)